app/metrics/utils.go

```diff
@@ -0,0 +1,119 @@
+package metrics
+
+import (
+	"time"
+
+	"github.com/cosmos/cosmos-sdk/telemetry"
+	gometrics "github.com/hashicorp/go-metrics"
+)
+
+type Label = gometrics.Label
+
+// NewLabel creates new Label with given name and value.
+func NewLabel(name, value string) Label {
+	return Label{Name: name, Value: value}
+}
+
+// ModuleMeasureSinceWithLabels emits a time measure metric for a module with a given set of keys and labels.
+func ModuleMeasureSinceWithLabels(module string, start time.Time, keys []string, extraLabels []Label) {
```

Comment:
question: Why do these first two functions `ModuleMeasureSinceWithLabels` and `ModuleIncrCounterWithLabels` not start with the same telemetry enabled check?